src/devices/xiaomi.ts

```diff
@@ -2558,7 +2654,80 @@ const definitions: Definition[] = [
         exposes: [e.switch().withEndpoint('l1'), e.switch().withEndpoint('l2'),
             e.power(), e.current(), e.energy(), e.voltage(), e.device_temperature(),
         ],
-        extend: [xiaomi.extend.switchType],
+        extend: [
+            xiaomi.extend.switchType,
+            extend.binary({
+                name: 'interlock',
+                valueOn: true,
+                valueOff: false,
+                cluster: 'aqaraOpple',
+                attribute: {id: 0x02d0, type: 0x10},
+                description: 'Enabling prevents both relays being on at the same time (Interlock)',
+                zigbeeCommandOptions: {manufacturerCode},
+            }),
+            extend.enumLookup({
+                name: 'power_on_behavior',
+                lookup: {'on': 0, 'previous': 1, 'off': 2, 'reverse': 3},
+                cluster: 'aqaraOpple',
+                attribute: {id: 0x0517, type: 0x20},
+                description: 'Controls the behavior when the device is powered on after power loss',
+                zigbeeCommandOptions: {manufacturerCode},
+            }),
+            extend.enumLookup({
+                name: 'operation_mode',
+                lookup: {'decoupled': 0, 'control_relay': 1},
+                cluster: 'aqaraOpple',
+                attribute: {id: 0x0200, type: 0x20},
+                description: 'Decoupled mode for 1st relay',
+                zigbeeCommandOptions: {manufacturerCode},
+                endpoint: 'l1',
+            }),
+            extend.enumLookup({
+                name: 'operation_mode',
+                lookup: {'decoupled': 0, 'control_relay': 1},
+                cluster: 'aqaraOpple',
+                attribute: {id: 0x0200, type: 0x20},
+                description: 'Decoupled mode for 2nd relay',
+                zigbeeCommandOptions: {manufacturerCode},
+                endpoint: 'l2',
+            }),
+            extend.enumLookup({
+                name: 'mode',
+                lookup: {'power': 0, 'pulse': 1, 'dry': 3},
+                cluster: 'aqaraOpple',
+                attribute: {id: 0x0289, type: 0x20},
+                description: 'Work mode: Power mode, Dry mode with impulse, Dry mode',
+                zigbeeCommandOptions: {manufacturerCode},
+            }),
+            extend.numeric({
+                name: 'pulse_length',
+                valueMin: 200,
+                valueMax: 2000,
+                unit: 'ms',
+                cluster: 'aqaraOpple',
+                attribute: {id: 0x00eb, type: 0x21},
+                description: 'Impulse length in Dry mode with impulse',
+                zigbeeCommandOptions: {manufacturerCode},
+            }),
+            extend.enumLookup({
```

Comment:
I didn't think about this case yet, but I believe it generates 2 action exposes which is not expected